capture.py

```diff
@@ -126,9 +136,10 @@ def capture(camera=None,
 
         with contextlib.nested(
              _maintain_camera(panel, camera),
-             _applied_viewport_options(viewport_options, panel),
```

Comment:
Ok, I'm not criticising, just want to understand the reason for the change. So do you think the previous approach should stick around?